exercises/practice/food-chain/food_chain_test.go

```diff
@@ -86,21 +87,23 @@ func diff(got, want string) string {
 
 func TestVerse(t *testing.T) {
 	for v := 1; v <= 8; v++ {
-		if ret := Verse(v); ret != ref[v] {
-			t.Fatalf("Verse(%d) =\n%s\n  want:\n%s\n%s", v, ret, ref[v], diff(ret, ref[v]))
-		}
+		t.Run(strconv.Itoa(v), func(t *testing.T) {
```

Comment:
I think we can put a description here too. In the simplest form, it can just be an `strconv.Itoa(v) + " verses"`, but it might be worth to convert the test cases to a slice of test cases, where each test case can be something like:

```go
type testCase struct {
    description string
    verses int
    expectedText string
}
```

Reply:
To be honest I don't think there is a need of adding a struct and testcases, I have simply added `verse` to the description, what do you think?